packages/eslint-plugin/typings/eslint-rules.d.ts

```diff
@@ -142,6 +142,69 @@ declare module 'eslint/lib/rules/indent' {
   export = rule;
 }
 
+declare module 'eslint/lib/rules/keyword-spacing' {
+  import { TSESLint, TSESTree } from '@typescript-eslint/experimental-utils';
+
+  const rule: TSESLint.RuleModule<
+    'expectedBefore' | 'expectedAfter' | 'unexpectedBefore' | 'unexpectedAfter',
+    [],
+    {
+      // Statements
+      DebuggerStatement: (node: TSESTree.DebuggerStatement) => void;
```

Comment:
You could type these using `TSESLint.RuleFunction<T>` to tie them back to the single type, but that might be overkill.

It's probably worth making these methods however, to be consistent with the other types.